aspnetcore/mvc/models/file-uploads/samples/3.x/SampleApp/Program.cs

```diff
@@ -20,6 +20,12 @@ public static IHostBuilder CreateHostBuilder(string[] args) =>
                 .ConfigureWebHostDefaults(webBuilder =>
                 {
                     webBuilder.UseStartup<Startup>();
+                    webBuilder.ConfigureKestrel(options =>
+                    {
+                        options.Limits.MaxRequestBodySize = 6L << 30; // 6 GB
```

Comment:
This actually matters.  The next two should just help perf for HTTP/2 (always measure though).